builder/builder.go

```diff
@@ -58,7 +62,14 @@ func (builder *Builder) name() string {
 }
 
 func (builder *Builder) option() string {
-	return fmt.Sprintf("--with-%s", builder.name())
+	name := builder.name()
+
+	// only libressl does not match option name
+	if name == "libressl" {
+		name = "openssl"
```

Comment:
Only libressl does not match option name and software name. So I rewritten the option name here.